specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/virtualMachineScaleSet.json

```diff
@@ -4144,11 +4144,6 @@
       "required": [
         "name"
       ],
-      "allOf": [
-        {
-          "$ref": "../../../common-types/v1/common.json#/definitions/SubResource"
-        }
-      ],
```

Comment:
Removing this introduces quite a tricky breaking change in dotnet SDK. We need to discuss about this

Reply:
resolving this since we have offline talked